pch_mgr.cpp

```diff
@@ -99,17 +98,22 @@ const char *ResourceManager::get_file(const char *path, bool binary,
   return m_buffers[key].first;
 }
 
-const char* ResourceManager::realloc_buffer(const char *id,
-                                            const char* buf, size_t size,
-                                            bool requireNullTerminate) {
+void ResourceManager::realloc_buffer(const char *id,
+                                            const char* buf, size_t size) {
```

Comment:
Formatting around function arguments looks strange, could you please fix it?

Reply:
Fixed